docs/contributing.rst

```diff
@@ -133,6 +140,31 @@ To run the skipped tests for the whole terminal session, execute::
 
 and then run the tests as usual.
 
+Running tests with large files
+------------------------------
+
+To run the tests with large input or output files, ensure you have the
+`Git Large File Storage`_ extension installed and run the tests as usual.
+
+Adding tests with large files
+-----------------------------
+
+If a test file is larger than 1 MiB, then put it in the
+``resolwe_bio/tests/processes/files/large/`` directory. Git Large File Storage
+(LFS) extension will automatically pick it up and treat it appropriately.
+
+To ensure contributors without Git LFS or users using the source distribution
+can smoothly run the tests, decorate the tests using large files with the
+following::
+
+    @unittest.skipUnless(*check_large_files(<large-file1>, <large-file2>, ...))
```

Comment:
This is a bit ugly, we could write a custom decorator, for instance `@skip_large_files(<large-file1>, <large-file2>, ...)`. Just an idea, not required for this PR.